### ISSUE TITLE
Switch off automation by default

### ISSUE THREAD

**Issue by ricksbrown** [BorderTech/wcomponents#1088]
To minimize the amount of client side code we should conditionally load the automation code.

Probably the best way to do this is by requiring automation testers to set a cookie before they request a page. If the cookie is present we will deliver the automation code, otherwise (the default) we won't.

**Comment by ricksbrown** [comment]
After experimenting with this for a while I've found that it is difficult to work with cookies on Selenium. This is because you can only set a cookie for the domain you are on.

If you load the page and then set the cookie it's too late.
If you try to set the cookie before you load the page you have domain issues.

**Comment by marksreeves** [comment]
The automation module is itself quite light and has dependencies only on other commonly used modules. We tweaked load in 1.4.0 to not load the automation if no JavaScript is loaded, I could further tweak this to not load a8n if there is no Ajax (eager, datalists, WAjaxControl, whatever) but that is probably over-kill. 

I really think this is a bit of a none-issue which was brought to light by our previously rather lax JS loader which resulted in modules leaking out of common.js. We have fixed that double load problem so I think we can just ignore this to be honest. Opinions @ricksbrown ?
